File: Java Barcode Scanner ME/app/src/main/java/de/dimedis/mobileentry/backend/response/ServerConnectResponseContent.java
package de.dimedis.mobileentry.backend.response;

import com.google.gson.annotations.SerializedName;

import java.util.Map;

public class ServerConnectResponseContent extends BaseResponseContent {
    @SerializedName("server_name")
    public String serverName;
    @SerializedName("rpc_url")
    public String rpcUrl;
    public Map<String, String> languages;

    @Override
    public String toString() {
        return "ServerConnectResponseContent{" +
                "serverName='" + serverName + '\'' +
                ", rpcUrl='" + rpcUrl + '\'' +
                ", languages=" + languages +
                '}';
    }
}

File: Kotlin Migration/NTMobileEntry/app/src/main/java/de/dimedis/mobileentry/backend/response/ServerConnectResponseContent.kt
package de.dimedis.mobileentry.backend.response

import com.google.gson.annotations.SerializedName

class ServerConnectResponseContent : BaseResponseContent() {
    @SerializedName("server_name")
    val serverName: String? = null

    @SerializedName("rpc_url")
    val rpcUrl: String? = null
    val languages: Map<String, String>? = null

    override fun toString(): String {
        return "ServerConnectResponseContent{" +
                "serverName='" + serverName + '\'' +
                ", rpcUrl='" + rpcUrl + '\'' +
                ", languages=" + languages +
                '}'
    }
}